soh/soh/Enhancements/randomizer/3drando/shops.cpp

```diff
@@ -128,24 +128,20 @@ static constexpr std::array<double, 60> ShopPriceProbability= {
   0.959992180, 0.968187000, 0.975495390, 0.981884488, 0.987344345, 0.991851853, 0.995389113, 0.997937921, 0.999481947, 1.000000000,
 };
 
-std::map<uint8_t, int> affordableCaps = {
-    {RO_SHOPSANITY_PRICE_STARTER, 10},
-    {RO_SHOPSANITY_PRICE_ADULT,   105},
-    {RO_SHOPSANITY_PRICE_GIANT,   205},
-    {RO_SHOPSANITY_PRICE_TYCOON,  505},
-};
-
-// If affordable option is on, cap items at affordable price just above the max of the previous wallet tier
-int CapPriceAffordable(int value, int cap) {
-    if (Settings::ShopsanityPricesAffordable.Is(true) && value > cap)
-        return cap;
-    return value;
-}
+std::vector<int> priceCaps = { 10, 105, 205, 505 };
```

Comment:
this isn't being used outside of `GetPriceAffordable`, and seems to never be modified, might make sense to move it into `GetPriceAffordable` and make it `static const`

also, `priceCaps` doesn't feel like a reasonable name given it's use. I'd prefer `affordablePrices` or something like that

Reply:
Name changes and static const make sense.